### PR TITLE
Exporting genFont and adjustFont

### PR DESCRIPTION
**Pull request by suresh-gangumalla**
exporting genFont and adjustFont functionalites to support msdf-generator as library alongside CLI.

made few code changes to fulfil library requirements

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,8 +27,11 @@ export async function generateFonts() {
     for (const ext of font_exts) {
       if (file.endsWith(ext)) {
         fontsFound++;
-        await adjustFont(await genFont(file, 'msdf'));
-        await adjustFont(await genFont(file, 'ssdf'));
+        let font = await genFont(file, 'msdf');
+        if (font) await adjustFont(font);
+
+        font = await genFont(file, 'ssdf')
+        if (font) await adjustFont(font);
```
Comment (michielvandergeest) [inline src/index.ts:34]: I would personally not reuse the `let font` variable, but be more explicit and name the first one `const msdfFont` and the second `const ssdfFont` (or something like that).

**Review by wouterlucas — APPROVED** [review]
LGTM!

File: src/index.ts (hunk 1 above)
Comment (suresh-gangumalla) [reply src/index.ts]: @michielvandergeest  agreed. Updated variable names to descriptive name

**Review by michielvandergeest — APPROVED** [review]
LGTM

**Pull request merged**